crates/fuel-data-parser/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "fuel-data-parser"
+authors = { workspace = true }
+keywords = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+version = { workspace = true }
+
+[dependencies]
+anyhow = { workspace = true }
+
+# compression
+async-compression = { version = "0.4.11", features = ["all"] }
+async-nats = { workspace = true }
+async-trait = { workspace = true }
+bincode = "1.3.3"
+
+# ser/deser
+bytes = "1.6.0"
```

Comment:
can you pass here dependencies that are already being used in other crates to the workspace?

Reply:
sure, I have cleaned it all up.